pkgs/by-name/ei/eiffel-studio-bin/package.nix

```
@@ -0,0 +1,70 @@
+{
+  autoPatchelfHook,
+  gcc,
+  gnumake,
+  gtk2-x11,
+  lib,
+  makeWrapper,
+  stdenv,
+  xorg,
+  build ? "103187",
+  libraries ? [ ],
+  platform ? "linux-x86-64",
+  version ? "19.05",
+}:
+
+stdenv.mkDerivation {
+  pname = "eiffelstudio-bin";
+  version = version;
+  src = fetchTarball {
```

nope, you must use Nixpkgs fetchers, that fetch on build and not on eval
```suggestion
  src = fetchzip {
```